### ISSUE TITLE
memo: New does not work on 0.5.3

### ISSUE THREAD

**Issue by satokaz** [satokaz/vscode-memo-life-for-you#55]
```
rejected promise not handled within 1 second: Error: EISDIR: illegal operation on a directory, read
extensionHostProcess.js:1048
stack trace: Error: EISDIR: illegal operation on a directory, read
	at Object.readSync (fs.js:506:3)
	at tryReadSync (fs.js:331:20)
	at Object.readFileSync (fs.js:368:19)
	at Object.readFileSync (electron/js2c/asar.js:542:40)
	at memoNew.memoTemplate (/Users/kazus/Gits/vscode-memo-life-for-you/dist/extension.js:34797:28)
	at /Users/kazus/Gits/vscode-memo-life-for-you/dist/extension.js:34682:38
	at processTicksAndRejections (internal/process/task_queues.js:94:5)
```

**Comment by satokaz** [comment]
https://github.com/satokaz/vscode-memo-life-for-you/blob/b0fb568390d424148e6311de417111ab60d86d30/src/memoConfigure.ts#L123-L125

If the path is a zero-length string, '.' is returned, representing the current working directory.

**Comment by satokaz** [comment]
Fix #56
